src/hotspot/cpu/x86/nativeInst_x86.cpp

```diff
@@ -51,15 +51,6 @@ void NativeCall::verify() {
   }
 }
 
-address NativeCall::destination() const {
-  // Getting the destination of a call isn't safe because that call can
-  // be getting patched while you're calling this.  There's only special
-  // places where this can be called but not automatically verifiable by
-  // checking which locks are held.  The solution is true atomic patching
-  // on x86, nyi.
-  return return_address() + displacement();
-}
-
```

Comment:
Is this change included by accident?

Reply:
Ouch. This crept into my workspace, sorry. Reverted.